package.json

```diff
@@ -128,7 +128,7 @@
   "dependencies": {
     "@aws-crypto/client-node": "^3.1.1",
     "@elastic/datemath": "5.0.3",
-    "@elastic/eui": "npm:@opensearch-project/oui@1.3.0",
+    "@elastic/eui": "npm:@opensearch-project/oui@1.4.0-alpha.2",
```

Comment:
I don't think we want alpha versions in `2.x` branch? In `main` it makes sense, but we should keep `2.x` releasable.

Reply:
+ 1

Reply:
This is temporary until OUI 1.4 is released.

Reply:
Yeah, but it makes the 2.x branch unreleasable, which I thought we wanted to avoid.